addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7296,12 +7296,12 @@ msgid "De-interlace (Half)"
 msgstr ""
 
 msgctxt "#16037"
-msgid "Deinterlace video"
+msgid "De-interlace video"
```

Comment:
In general you only add a `-` in English if there would otherwise be two of the same vowels in a row, e.g. you write "de-emphasize", not "deemphasize".